packages/core/protocols/src/proto/dxos/config.proto

```diff
@@ -95,15 +101,24 @@ message Runtime {
     /// Milliseconds
     optional int32 invitation_expiration = 5;
 
-    // TODO(burdon): Remove?
     optional string remote_source = 6;
   }
 
   message App {
+    message BuildInfo {
+      /// RFC-3339 datetime string.
+      optional string build_time = 1;
+      optional string version = 2;
+      optional string commit_hash = 3;
+    }
+
```

Comment:
need `app_id`, `org_id`

theme feels like part of a sub `options` message

similarly website, public_url as part of a deployment message

Reply:
afaik none of those are being used currently so we could probably redo that section later